pkg/test/framework/components/environment/kube/flags.go

```diff
@@ -239,7 +285,7 @@ func init() {
 		"", "Specifies the mapping for each cluster to the cluster hosting its config. The value is a "+
 			"comma-separated list of the form <clusterIndex>:<configClusterIndex>, where the indexes refer to the order in which "+
 			"a given cluster appears in the 'istio.test.kube.config' flag. If not specified, the default is every cluster maps to itself(e.g. 0:0,1:1,...).")
-	flag.StringVar(&topologyFile, "istio.test.kube.topology", "", "The path to a JSON file that defines control plane,"+
+	flag.Var(&clusterConfigs, "istio.test.kube.topology", "The path to a JSON file that defines control plane,"+
```

Comment:
nit: do we need to update the flag description?

Reply:
For now, I'm gonna leave the flag descriptions as is. It might make sense to have a custom replacement for `flag.Var` that behaves the same but also stores a `usage` for our custom config file.